BLSQ/iaso
# Fix IA-1632 Fix bug in  previous migration to delete beneficiary

Previous migration was failing when user with the perm existed Also the permission got created again because it was still listed as a permission


Explain what problem this PR is resolving

## Changes

1. Disable previous migration
2. Migration to Remove perm from user that have them
3. Remove the perm from the python code, migration to sync
4. Migration to remove the existing permissions

## How to test

* Add perms to user (before switching to this branch to avoid running the migration per error)
* Run migrations
* Check that the beneficiary permission is no longer listed in the menu

## Notes
I didn't check for group permission as we normally don't use group in Iaso, shouldn't be a problem
